Makefile

```diff
@@ -5,6 +5,13 @@ NOSTRETCH ?= 0
 NOBUSTER ?= 0
 NOBULLSEYE ?= 0
 
+export Q ?= @
+ifneq ($(QUIET),)
```

Comment:
Can this be changed to `ifeq ($(QUIET),1)`? Setting `QUIET=0` shouldn't make it quiet.

Reply:
Sure, I will update the change